Sources/OpenTelemetrySdk/Resources/ResourceAttributes.swift

```diff
@@ -253,48 +247,117 @@ public enum ResourceAttributes: String {
     */
     case deploymentEnvironment = "deployment.environment"
     /**
-    The name of the function being executed.
+    A unique identifier representing the device.
+
+    ~~~
+    // Examples
+    attributes[.deviceId] = "2ab2916d-a51f-4ac8-80ee-45ac31a28092"
+    ~~~
+
+    - Note: The device identifier MUST only be defined using the values outlined below. This value is not an advertising identifier and MUST NOT be used as such. On iOS (Swift or Objective-C), this value MUST be equal to the [vendor identifier](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor). On Android (Java or Kotlin), this value MUST be equal to the Firebase Installation ID or a globally unique UUID which is persisted across sessions in your application. More information can be found [here](https://developer.android.com/training/articles/user-data-ids) on best practices and exact implementation details. Caution should be taken when storing personal data or anything which can identify a user. GDPR and data protection laws may apply, ensure you do your own due diligence.
+
+    - Requires: Value type should be `String`
+    */
+    case deviceId = "device.id"
+    /**
+    The model identifier for the device.
+
+    ~~~
+    // Examples
+    attributes[.deviceModelIdentifier] = "iPhone3,4"
+    attributes[.deviceModelIdentifier] = "SM-G920F"
+    ~~~
+
+    - Note: It's recommended this value represents a machine readable version of the model identifier rather than the market or consumer-friendly name of the device.
+
+    - Requires: Value type should be `String`
+    */
+    case deviceModelIdentifier = "device.model.identifier"
```

Comment:
These are the new attributes that we discussed in a SIG meeting a couple months back. Will want to add them (in a future PR) to the resource package.

Reply:
I made an issue https://github.com/open-telemetry/opentelemetry-swift/issues/274